jaeger/jaeger_install/rhbjaeger-architecture.adoc

```diff
@@ -0,0 +1,44 @@
+[id="jaeger-architecture"]
+= Jaeger architecture
+include::modules/jaeger-document-attributes.adoc[]
+:context: jaeger-architecture
+
+toc::[]
+
+Every time a user takes an action in an application, a request is executed by the architecture that may require dozens of different services to participate in order to produce a response.
+Jaeger lets you perform distributed tracing, which records the path of a request through various microservices that make up an application.
+
+_Distributed tracing_ is a technique that is used to tie the information about different units of work together—usually executed in different processes or hosts—in order to understand a whole chain of events in a distributed transaction.
+Distributed tracing lets developers visualize call flows in large microservice architectures.
+It can be invaluable in understanding serialization, parallelism, and sources of latency.
```

Comment:
What if we put developers first in this sentence since it's directed at them. I also think that invaluable is a superlative and I'm not sure it goes with 'can' there. I think it's either invaluable or it's not; it's not possible that it's invaluable. 

I think it should be either it is invaluable or it is valuable or it can be valuable.

Developers can visualize call flows in large microservice architectures with distributed tracing. It's valuable for understanding serialization, parallelism, and sources of latency.